vocab/wd/index-linktemplate.html

```diff
@@ -1520,6 +1520,25 @@
   <h3>Named Individuals</h3>
 
 <section class="term">
+  <h4>assessing</h4>
+  <p>The motivation for when the user intends to provide an assessment about the Target resource.</p>
+  <div class="tech">
+    <ul>
+      <li><strong>URI:</strong> http://www.w3.org/ns/oa#assessing</li>
+      <li><strong>Instance Of:</strong> oa:Motivation</li>
+    </ul>
+  </div>
+  <div>
+    <pre class="example highlight" title="oa:assessing">
+&lt;http://example.org/anno%%anno%%&gt; a oa:Annotation ;
+    oa:hasBody [ rdf:value "Very high quality" ] ;

```

Comment:
oa:hasBody [ rdf:value "Very high quality" ] -> oa:hasBody [ a oa:TextualBody ; rdf:value "Very high quality" ]
Except for example 18 in the data model and a few examples like this one in the vocab doc about motivation/purpose and about dctypes properties, our examples conform to 3.2.4, 'The Body SHOULD have [type] TextualBody class...'  Should we be consistent about following our own SHOULD in these examples?